### PR TITLE
Status check for reverting a commit

### PR DESCRIPTION
**Pull request by Mariatta**
If a PR starts with the word "Revert", then check if the
body contains the reason for the revert.

Closes https://github.com/python/core-workflow/issues/72

### CODE REVIEW

File: bedevere/revert_pr.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import re
+
+from . import router
+
+
+REVERT_RE = re.compile(r"^Revert")
+STATUS_TEMPLATE = {"context": "bedevere/revert-pr"}
```
Comment (brettcannon) [inline bedevere/revert_pr.py]: This is too generic of a context. Maybe "bedevere/reason-to-revert"?

File: bedevere/revert_pr.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+SUCCESS_STATUS = STATUS_TEMPLATE.copy()
+SUCCESS_STATUS["state"] = "success"
+SUCCESS_STATUS["description"] = "Found the reason to revert the PR."
```
Comment (brettcannon) [inline bedevere/revert_pr.py]: "Reason for reverting the commit is specified."?

**Comment by Mariatta** [comment]
Feel free to close if it's decided that we don't need a status check for this :) Thanks.

**Comment by brettcannon** [comment]
Thanks for at least taking a stab at this, @Mariatta 😄

**Pull request closed without merging**